src/etalab-2.0.xml

```diff
@@ -0,0 +1,196 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="etalab-2.0" name="Etalab Open License 2.0" listVersionAdded="3.7">
+      <crossRefs>
+         <crossRef>https://github.com/DISIC/politique-de-contribution-open-source/blob/master/LICENSE.pdf</crossRef>
+         <crossRef>https://raw.githubusercontent.com/DISIC/politique-de-contribution-open-source/master/LICENSE</crossRef>
+      </crossRefs>
+    <notes>English translation can be found here: https://www.etalab.gouv.fr/wp-content/uploads/2018/11/open-licence.pdf</notes>
+    <text>
+
+      <titleText>
+            <p>LICENCE OUVERTE / OPEN LICENCE</p>
+            <optional><p>===================================================================</p></optional>
```

Comment:
if the TitleText is optional anyway, do we need the separate optional tags? or at least  just use one? Seems weird to have optional tag inside a TitleText tag in any case??
@swinslow

Reply:
Good question, and I don’t know the right answer... here is a recent one where we included optional tags within titleText: https://github.com/spdx/license-list-XML/blob/master/src/BlueOak-1.0.0.xml

I’m okay with it either way... I am assuming it should be disregarded by matching guidelines either way, so I imagine it wasn’t necessary to add them for the various lines in this one =)